src/lib.rs

```diff
@@ -1434,8 +1432,6 @@ impl_zeroable! {
     //
     // In this case we are allowed to use `T: ?Sized`, since all zeros is the `None` variant.
     {<T: ?Sized>} Option<NonNull<T>>,
```

Comment:
Adding a note that we should not forget to do the equivalent fix to https://github.com/Rust-for-Linux/linux/commit/df27cef153603b18a7d094b53cc3d5264ff32797 here:

```suggestion
    {<T>} Option<NonNull<T>>,
```